elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -204,6 +202,43 @@ class PfeAutocomplete extends PFElement {
     this.setAttribute("debounce", val);
   }
 
+  _slotchangeHandler() {
+    // input box
+    let slotNodes = this.shadowRoot.querySelector("slot").assignedNodes();
+    let slotElems = slotNodes.filter(n => n.nodeType === Node.ELEMENT_NODE);
+
+    if (slotElems.length === 0) {
+      console.warn(
+        `${PfeAutocomplete.tag}: There must be a input tag in the light DOM`
+      );
+
+      return;
+    }
+
+    this._input = slotElems[0];
+
+    if (this._input.tagName.toLowerCase() !== "input") {
+      console.warn(
+        `${PfeAutocomplete.tag}: The only child in the light DOM must be an input tag`
```

Comment:
Since it's only a warning and not an error, should we update the language of the message to `should be an input tag`?

Reply:
@castastrophe - Should we change to `console.warning`? The whole thing doesn't work without an input tag.

Reply:
Should it just throw an error and fail to upgrade?